src/common/util/include/openvino/util/common_util.hpp

```diff
@@ -14,6 +14,18 @@
 namespace ov {
 namespace util {
 
+// helper struct to tell wheter type T is any of given types U...
+// termination case when U... is empty -> return std::false_type
+template <class T, class... U>
```

Comment:
template <class...> should suffice as neither T nor U is used

Reply:
Currently, this PR doesn't change this file.